src/scene/materials/standard-material-options-builder.js

```diff
@@ -63,6 +63,12 @@ class StandardMaterialOptionsBuilder {
 
     _updateSharedOptions(options, scene, stdMat, objDefs, pass) {
         options.forceUv1 = stdMat.forceUv1;
+
+        // USER ATTRIBUTES
+        if (stdMat.userAttributes) {
+            options.litOptions.userAttributes = Object.fromEntries(stdMat.userAttributes.entries());
```

Comment:
All is good now, just one little thing - could you please add 'userAttributes' member to the lit-shader-options.js as well.